From 4c4056953b4369742d052d01e9f4e20334fa0d68 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Mon, 25 Jul 2016 09:50:12 +0800
Subject: [PATCH] regulator: lp8752: set dcdc mode as auto mode

lp8752 static power is too high when in foce pwm mode,
change it's mode to auto.

Change-Id: I6b1342cc8b63d240d90194b52f5926501ea7babf
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/regulator/lp8752.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/regulator/lp8752.c b/drivers/regulator/lp8752.c
index 0062dee31a59..ea06d755f44d 100644
--- a/drivers/regulator/lp8752.c
+++ b/drivers/regulator/lp8752.c
@@ -260,10 +260,10 @@ static int lp8752_probe(struct i2c_client *client,
 
 	ret = regmap_update_bits(pchip->regmap,
 				 LP8752_CTRL_BUCK0,
-				 (1 << 0), 0x1);
+				 (1 << 0), 0);
 	ret = regmap_update_bits(pchip->regmap,
 				 LP8752_CTRL_BUCK2,
-				 (1 << 0), 0x1);
+				 (1 << 0), 0);
 
 	if (!pdata) {
 		pchip->pdata = devm_kzalloc(pchip->dev,
-- 
2.35.3

